dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -0,0 +1,243 @@
+/*******************************************************************************
+ *     HPCC SYSTEMS software Copyright (C) 2019 HPCC Systems®.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *******************************************************************************/
+package org.hpccsystems.dfs.client;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import org.hpccsystems.dfs.client.HPCCFile;
+import org.hpccsystems.dfs.client.HPCCRecord;
+import org.hpccsystems.dfs.client.HPCCRecordBuilder;
+import org.hpccsystems.dfs.client.HPCCRecordAccessor;
+import org.hpccsystems.dfs.client.HpccRemoteFileReader;
+import org.hpccsystems.dfs.client.ReflectionRecordBuilder;
+import org.hpccsystems.dfs.client.DataPartition;
+
+import org.hpccsystems.dfs.cluster.*;
+
+import org.hpccsystems.commons.ecl.FieldDef;
+import org.hpccsystems.commons.ecl.RecordDefinitionTranslator;
+import org.hpccsystems.ws.client.utils.Connection;
+import org.hpccsystems.ws.client.HPCCWsDFUClient;
+import org.hpccsystems.ws.client.wrappers.wsdfu.DFUCreateFileWrapper;
+import org.hpccsystems.ws.client.wrappers.wsdfu.DFUFilePartWrapper;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DFSReadWriteTest
+{
+
+    private static final String clusterIP = "192.168.56.101";
```

Comment:
can we attempt to read sys properties to populate the protocol, ip, port, user/pass?